task2/src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,66 @@
+import { Header, Main, Footer, AppTitle } from '@components';
+import { MovieCatalog } from './components/MovieCatalog/MovieCatalog';
+import { AddMovieButton } from './components/AddMovieButton/AddMovieButton';
+import styled from 'styled-components';
+import { MovieSearch } from './components/MovieSearch/MovieSearch';
+import { MovieSort } from './components/MovieSort/MovieSort';
+
+const MarginWrapperStyled = styled.div`
+    margin-top: 10px;
+    margin-bottom: 30px;
+`;
+
+const PaddingWrapperStyled = styled(MarginWrapperStyled)`
+    padding: 10px 50px;
+`;
+
+const HeaderTopStyled = styled(MarginWrapperStyled)`
+    position: relative;
+    display: flex;
+	flex-direction: row;
```

Comment:
check indents everywhere

install formatter to your IDE

Reply:
Sorry, Default VS Code formatter does not support styled-components strings.